features/admin/pod.feature

```diff
@@ -218,7 +218,7 @@ Feature: pod related features
       | p | {"metadata":{"annotations": {"openshift.io/node-selector": ""}}}|
     Then the step should succeed
     Given I store the schedulable nodes in the :nodes clipboard
-    Given environment has at least 2 schedulable nodes
+    Given environment has at least 2 schedulable worker nodes
```

Comment:
You can add your name to the author, for example `# @author weinliu@redhat.com` to count as a contributor.

Reply:
After this line https://github.com/openshift/verification-tests/blob/6bec9295c6fdb68e15432961490eba6ad06c671e/features/admin/pod.feature#L210